src/app/components/modules/simplified-lunch/simplified-lunch.component.scss

```diff
@@ -0,0 +1,269 @@
+	right: 0;
+	bottom: 0;
+	left: 0;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+
+	.loading {
+		width: 35rem;
+		padding: 1rem;
+		text-align: center;
+		color: $sophisticated-white;
+
+		h1 {
+			font-size: 3rem;
+			font-weight: 300;
+		}
+	}
+}
+
+.lunch-container {
+	padding: 2rem 0 0 0;
+	margin-bottom: 6rem;
+	.school-selection {
+		margin-bottom: 1rem;
+		text-align: center;
+
+		.btn {
+			font-size: small;
+		}
+	}
+
+	.week-navigation {
+		margin-bottom: 0.5rem;
+		display: flex;
+
+		.previous-week,
+		.current-week,
+		.next-week {
+			width: 100%;
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			padding: 1rem 0 1rem;
+
+			border: none;
+			color: $sophisticated-white;
+
+			&:hover {
+				color: darken($sophisticated-white, 20%);
+			}
+
+			&:active {
+				color: darken($sophisticated-white, 40%);
+			}
+		}
+
+		.previous-week,
+		.next-week {
+			font-size: 1rem;
+			font-weight: 300;
+
+			.fa {
+				margin: 0 0.5rem;
+			}
+		}
+
+		.current-week {
+			display: none;
+			font-size: 1rem;
+			font-weight: 600;
+			border-left: 1px solid rgba($sophisticated-white, 0.5);
+			border-right: 1px solid rgba($sophisticated-white, 0.5);
+		}
+	}
+
+	.lunch-week {
+		display: flex;
+		justify-content: space-between;
+		width: auto;
+		height: auto;
+		overflow-x: scroll;
+
+		.lunch-day {
+			position: relative;
+			flex: 0 0 100%;
+			padding: 0.5rem;
+			color: $sophisticated-white;
+
+			&.today {
+				border: 0.25rem solid $sophisticated-white;
+
+				&:after {
+					content: '';
+					position: absolute;
+					top: 0;
+					right: 0;
+					bottom: 0;
+					left: 0;
+					pointer-events: none;
+
+					background: rgba($sophisticated-white, 0.2);
+				}
+			}
+
+			.lunch-date {
+				margin-bottom: 1rem;
+
+				h3 {
+					margin-bottom: 0.25rem;
+					font-weight: 300;
+					letter-spacing: 0.03rem;
+				}
+
+				h5 {
+					margin: 0;
+				}
+			}
+
+			.lunch-menu {
+				.no-lunch {
+					padding: 1rem;
+					text-align: center;
+					border-top: 1px solid rgba(white, 0.5);
+
+					p {
+						margin: 0;
+						font-size: 1.25rem;
+						font-weight: 300;
+					}
+				}
+
+				.lunch .lunch-category {
+					padding-top: 0.5rem;
+
+
+					h3 {
+						text-align: center;
+						font-weight: 600;
+						letter-spacing: 0.03rem;
+					}
+
+					ul {
+						margin: 0 0 1rem 0;
+						padding: 0;
+						list-style: none;
+
+						li {
+							font-weight: 300;
+
+							&:before {
+								padding-right: 0.5em;
+								// Default bullet point
+								content: '🍔';
+							}
+
+							&.main-dish:before {
+								content: '🍔';
+							}
+
+							&.pizza-station:before {
+								content: '🍕'
+							}
+
+							&.soup:before {
+								content: '🍲'
+							}
+
+							&.dessert:before {
+								content: '🍩'
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+}
+
+
+
+/* --------------------------
+  Media Queries
+--------------------------- */
+
+@media screen and (min-width:544px) {
```

Comment:
For these homepage modules, we're only concerned about the module size itself and not the viewport's height/width. This should probably be moved to those custom css-element-queries (like the `.module-container` in lines 10, 18, and 26).

However some of these styles might not even be necessary since there's no navigation buttons to go forward/back a week.